### PR TITLE
test: add github workflow to auto-assign failed test issue to solid tests project

### PR DESCRIPTION
**Pull request by glorv**
Signed-off-by: glorv <glorvs@163.com>

###  What have you changed?
We have created a daily report job to find tests on our CI that fails occasionnally and auto create issures for them. This github workflow will auto assign these issues to `Solid Tests` project so we can manage them easier.

Similar workflow for tidb is [here](https://github.com/pingcap/tidb/blob/master/.github/workflows/assign_project.yml)

- Summarize your change

Don't assume reviewers understand the original issue.

###  What is the type of the changes?

- Misc (other changes)

### CODE REVIEW

File: .github/workflows/assign_failed_test_project.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    runs-on: ubuntu-latest
+    name: Assign to Solid Tests Project
+    steps:
+    - name: Run issues assignment to project Solid Tests Kanban
+      uses: srggrs/assign-one-project-github-action@1.2.0
+      if: |
+        github.event.issue.user.login == 'sre-bot' &&
+        contains(github.event.issue.labels.*.name, 'component/test-bench')
+      with:
+        project: 'https://github.com/pingcap/tidb/projects/14'
```
Comment (you06) [inline .github/workflows/assign_failed_test_project.yml]: It's odd to assign tidb project issues in tikv repo.

Comment (glorv) [reply .github/workflows/assign_failed_test_project.yml]: fixed, sorry for my carelsss fault

File: .github/workflows/assign_failed_test_project.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+jobs:
+  assign_bench_test_project:
+    runs-on: ubuntu-latest
+    name: Assign to Solid Tests Project
+    steps:
+    - name: Run issues assignment to project Solid Tests Kanban
+      uses: srggrs/assign-one-project-github-action@1.2.0
+      if: |
+        github.event.issue.user.login == 'sre-bot' &&
```
Comment (you06) [inline .github/workflows/assign_failed_test_project.yml]: Does it mean only issues created by bot will be added into the target project?

Comment (glorv) [reply .github/workflows/assign_failed_test_project.yml]: yes. Issues create by sre-bot with `bench-test` label are done by our daily report job, so these issues can be auto-added to the `Solid Tests`. I'm not sure if there are some other conditions, we should also do this. @mahjonp PTAL

Comment (BusyJay) [reply .github/workflows/assign_failed_test_project.yml]: Why can't bot do it directly?

**Pull request merged**